src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -122,6 +122,20 @@ export type Tag = {
   topValues: Array<TagTopValue>;
 };
 
+type PickTagParamsType = {
+  query: string;
+  field?: string[] | undefined;
+  project?: string | string[] | undefined;
+  sort?: string | string[] | undefined;
+  per_page?: number | undefined;
```

Comment:
What about the date range and environment parameters that come from global selection?